### PR TITLE
Add Winging It 7 & 8

### PR DESCRIPTION
**Pull request by SondraE**
## Related Issue(s)
Add Winging It Live 7 & 8 [#653](https://github.com/oddbird/oddleventy/issues/653)

## Steps to test/reproduce
https://deploy-preview-654--oddleventy.netlify.app

- /2024/05/16/winging-it-07/
- /2024/05/30/winging-it-08/
- posts list

### CODE REVIEW

File: content/blog/wingingit/winging-it-08.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+- [Inset Area](https://anchor-tool.com/)
+- [Polyfill](https://github.com/oddbird/css-anchor-positioning)
+
+## Demos:
+
+- [Anchor Size CodePen 1](https://codepen.io/jamessw/pen/dyENqQz)
+- [Anchor Size CodePen 2](https://codepen.io/jamessw/pen/OJYWorB)
+- [Popover](https://codepen.io/jamessw/pen/MWdjjQj)
+- [Anchor Scope](https://codepen.io/jamessw/pen/dyEXKeP)
+- [Fallback options with Position-try](https://codepen.io/jamessw/pen/abrZPQY)
```
Comment (jgerigmeyer) [inline content/blog/wingingit/winging-it-08.md:60]:
```suggestion
- [Fallback options with `position-try`](https://codepen.io/jamessw/pen/abrZPQY)
```

**Review by jgerigmeyer — CHANGES_REQUESTED** [review]

File: content/blog/wingingit/winging-it-07.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+) }}
+
+{{ utility.main_action(
+  'Subscribe to Channel »',
+  subscribe_url
+) }}
+
+## What we cover:
+
+- W
```
[the remark this answers is not in the document]
Comment (SondraE) [reply content/blog/wingingit/winging-it-07.md:42]: @stacyk Here's the PR where I could use your "What we cover" bullet points whenever you have a chance.

Comment (stacyk) [inline content/blog/wingingit/winging-it-07.md:42]:
```suggestion
- What are container queries and why you should use them
- How container queries differ from media queries
- Why some layouts collapse when you add containment
- Size containment with `contain-intrinsic-size`
- Why should we avoid adding containment on root
- Why a container can't query itself
- An opt-out strategy to remove containment
```

File: content/blog/wingingit/winging-it-07.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+## Links:
+
+- [Can We Query the Root Container?](https://www.oddbird.net/2023/07/05/contain-root/)
+- [Ahmad Shadeed's Interactive Guide to CSS Container Queries](https://ishadeed.com/article/css-container-query-guide/)
+- [A Cascading Layouts Workshop June 24-26, 2024](https://www.oddbird.net/workshops/cascading-layouts/)
```
[the remark this answers is not in the document]
Comment (stacyk) [reply content/blog/wingingit/winging-it-07.md:48]: on the youtube description, i updated this to show only the page if you want to do the same @SondraE 

```suggestion
- [Cascading Layouts Workshop](https://www.oddbird.net/workshops/cascading-layouts/)
```

**Comment by SondraE** [comment]
@jgerigmeyer Now all the content is in place. Ready for re-review.

**Review by stacyk — APPROVED** [review]

**Review by jgerigmeyer — APPROVED** [review]

**Pull request merged**